source/cloud/terraform.md

```diff
@@ -265,68 +263,69 @@ plan (only directly from OpenStack).
 :::{warning}
 Do not remove or modify the `port_something_ssh.json` files. These keep track of the external ports used. Deleting them _will_ break Terraform.
 :::
-## Making changes to module variables
-### Adding NFS or Nginx
-:::{tip}
-If you enabled these options upon initial creation of the VM, you don't have to do these steps
+## Automated variables
+The terraform templates include the ability to automatically adapt your VM based on certain variables.
+This is accomplished by installing scripts on the VM when it is first created.
+It requires ssh access to the VM.
+
+These features can be disabled by setting `scripts_enabled=false`.
+
+:::{warning}
+Automated variables will not work if you started using the templates before 11/2024.
```

Comment:
I would remove this from the docs as the previous warnings about the template changes. I would include this in the email for the current users not in the doc.